os/test/src/OpTests.scala

```diff
@@ -34,5 +34,15 @@ object OpTests extends TestSuite {
         os.remove(os.pwd / "out" / "scratch" / "nonexistent", checkExists = true)
       }
     }
+    test("pwd when home is not available") {
+      val correctPwd = java.nio.file.Paths.get(".").toAbsolutePath
+      val oldUserHome = System.getProperty("user.home")
+      System.setProperty("user.home", "?")
```

Comment:
This test doesn't test the behavior because the package object is already initialized when you call this method.
I tried running it on main and it passes.

Reply:
Right, then I guess there are no tests present that could capture that, I reverted the changes.

Reply:
I think having no unit tests are fine. We could go with an integration test in a separate test target that sets the `user.home` explicitly, but IMO this is a straightforward enough change to land as is